server/package.json

```diff
@@ -1,39 +1,40 @@
-{
-  "name": "server",
-  "version": "1.1.0",
-  "repository": "https://github.com/TeamShiksha/logoexecutive.git",
-  "license": "MIT",
-  "scripts": {
-    "start": "node app.js",
-    "lint": "eslint .",
-    "lint:fix": "eslint --fix .",
-    "test": "jest --watchAll=false",
-    "unit": "jest unit",
-    "coverage": "jest --coverage --watchAll=false"
-  },
-  "dependencies": {
-    "@aws-sdk/client-s3": "^3.472.0",
-    "@aws-sdk/cloudfront-signer": "^3.451.0",
-    "bcrypt": "^5.1.1",
-    "cookie-parser": "^1.4.6",
-    "cors": "^2.8.5",
-    "dayjs": "^1.11.10",
-    "dotenv": "^16.3.1",
-    "express": "^4.18.2",
-    "joi": "^17.12.3",
-    "jsonwebtoken": "^9.0.2",
-    "mongoose": "^8.5.1",
-    "morgan": "^1.10.0",
-    "multer": "^1.4.5-lts.1",
-    "nodemailer": "^6.9.7",
-    "uuid": "^9.0.1"
-  },
-  "devDependencies": {
-    "@shelf/jest-mongodb": "^4.3.2",
-    "@types/jest": "^29.5.6",
-    "eslint": "^8.46.0",
-    "jest": "^29.6.4",
-    "nodemon": "^3.0.1",
-    "supertest": "^6.3.3"
-  }
-}
+{
+  "name": "server",
+  "version": "1.1.0",
+  "repository": "https://github.com/TeamShiksha/logoexecutive.git",
+  "license": "MIT",
+  "scripts": {
+    "start": "node app.js",
+    "lint": "eslint .",
+    "lint:fix": "eslint --fix .",
+    "test": "jest --watchAll=false",
+    "unit": "jest unit",
+    "coverage": "jest --coverage --watchAll=false"
+  },
+  "dependencies": {
+    "@aws-sdk/client-s3": "^3.472.0",
+    "@aws-sdk/cloudfront-signer": "^3.451.0",
+    "bcrypt": "^5.1.1",
+    "cookie-parser": "^1.4.6",
+    "cors": "^2.8.5",
+    "dayjs": "^1.11.10",
+    "dotenv": "^16.3.1",
+    "express": "^4.18.2",
+    "handlebars": "^4.7.8",
+    "joi": "^17.12.3",
+    "jsonwebtoken": "^9.0.2",
+    "mongoose": "^8.5.1",
+    "morgan": "^1.10.0",
+    "multer": "^1.4.5-lts.1",
+    "nodemailer": "^6.9.7",
+    "uuid": "^9.0.1"
+  },
+  "devDependencies": {
+    "@shelf/jest-mongodb": "^4.3.2",
+    "@types/jest": "^29.5.6",
+    "eslint": "^8.46.0",
+    "jest": "^29.6.4",
+    "nodemon": "^3.0.1",
+    "supertest": "^6.3.3"
+  }
+}
```

Comment:
What is the change in package.json

Reply:
A new dependency handlebars is added for replacements in the HTML templates